### ISSUE TITLE
Can't create discussion

### ISSUE THREAD

**Issue by LuukPrins** [thedevdojo/chatter#237]
Everytime I try to create a new discussion the text-editor keeps loading as seen below. It didn't do this before and I don't know why it keeps loading. Please help. I also keep getting "$(...).spectrum is not a function" even though I've installed spectrum through NPM.

![image](https://user-images.githubusercontent.com/25355596/53325362-7436f080-38e3-11e9-8748-f731026f1bab.png)

**Comment by niloc7** [comment]
Yeah me too. Same problem. I think it's because it's not updated to the latest version of Laravel yet.

**Comment by LuukPrins** [comment]
If I in Chrome Devtools change the ID 'new_discussion' to something else to corrupt the css/js connected it does work (kind of). I think the loader is just bugged. Anyone any idea how to fix it?

**Comment by jgusta** [comment]
Hey everyone, this is a two part issue. I believe "$(...).spectrum is not a function" issue is a race condition due to jquery being loaded in app.js and this package trying to synchronously add the spectrum plugin to jquery. 

The horrible kludge I am using is to just synchronously load app.js by removing the "defer" attribute from `<script src="{{ asset('js/app.js') }}" defer></script>` in `resources/views/layouts/app.blade.php`. (thanks to https://github.com/thedevdojo/chatter/issues/226#issuecomment-438640209)

As for the second part see comment below.

Edit: Clarify scope of comment

**Comment by gitaudotdev** [comment]
If you're having issues with the creating new discussions check out 
👇
https://github.com/thedevdojo/chatter/issues/244#issuecomment-488680638
